### DISCUSSION TITLE
Removed Settings in a Git

### DISCUSSION THREAD

**Discussion by Jaydenwu719** [community/community#60960] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I Go to a Git (Any Git) I Can't Find Settings, What I am Trying to Do is to Create a Webhook, Please Reply. :)

**Comment by drahamim** [comment]
You need to have permissions for the repository to be able to see settings/ webhook configuration. If you do not see a Settings tab at the top of the repository (on the end of the row that Code/Insights/ other tabs are located) then you do not have permissions to modify any settings for the given repository.
